### PR TITLE
COMMERCE-9793 - Space added between quantity and 'In Stock'

### PR DESCRIPTION
**Pull request by brianikim**
From @elocamp,

> The problem was caused because there wasn't a space between the functions to return the quantity in stock and to get the language key for 'In Stock'. The proposed fix is just to add this missing space between the functions.
> 
> Related ticket: https://issues.liferay.com/browse/COMMERCE-9793

@andrea-ale-sbarra , 

low priority pull request.

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:commerce

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by victoroscuro** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/js/util/index.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ export function updateProductFields(data) {
 	].forEach(fillField);
 
 	stockQuantityContainer.innerHTML = cpInstance.stockQuantity
-		? cpInstance.stockQuantity + Liferay.Language.get('in-stock')
+		? cpInstance.stockQuantity + ' ' + Liferay.Language.get('in-stock')
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/js/util/index.js:58]: the test found that this is actually changing the label used in the single sku case. So please change as follow:
```suggestion
		? sub(Liferay.Language.get('x-in-stock'),  cpInstance.stockQuantity)
```

sub function is imported like this: _import sub from 'frontend-js-web';_ in other places

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
Requested change resolved @andrea-ale-sbarra

**Comment by MahmoudAzzam** [comment]
ci:test:sf

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by victoroscuro** [comment]
ci:test:commerce-functional

**Comment by victoroscuro** [comment]
ci:forward

**Pull request closed without merging**